### DISCUSSION TITLE
使用vc-collection-polyline动态渲染大量polyline的性能问题

### DISCUSSION THREAD

**Discussion by wide-sky-with-me** [zouyaoji/vue-cesium#298] [Q&A]
我时使用vue-cesium过程中遇到了一个需求，我需要每秒动态渲染大批量的polyline（大概300条）
`<vc-collection-polyline ref="polylineCollection" :polylines="polylines"></vc-collection-polyline>`
每一秒，我会改变polylines中的值，实现动态绘制，每秒绘制之前需要擦除之前的，我使用的是使用组件实例的reload方法，我也尝试了将组件实例的polylines属性置空进行擦除，但最后运行结果帧率只有10左右，我想询问下有没有更好的解决办法。

**Comment by zouyaoji** [comment]
注意2个点：
（1）升级到3.1.0版本。
（2）polylines中的polyline给一个唯一id。

如果没有改善请提供复现示例，我空了研究一下。
